finnflow/src/main/java/no/finn/android/finnflow/FinnFlow.java

```diff
@@ -21,30 +21,28 @@
     Screen destinationScreen = null;
     boolean isResumed = false;
 
-    public static Intent getIntent(Context context, Screen screen, Class<?> cls) {
-        return getIntent(context, History.single(screen), cls);
+    public FinnFlow(Activity activity, boolean debugBuild) {
```

Comment:
when initializing this in our code make sure you use FinnEnvironment.isDebug(). If we initialize this in a library we get the library's build config, not the "consumer" of the library.